force-app/test/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "eslint:recommended",
+    "plugins": ["jasmine"],
+    "env": {
+        "node": true,
+        "es2021": true,
+        "jasmine": true
+    },
+    "globals": {
+        "utam": "readonly",
+        "browser": "readonly"
+    },
+    "parserOptions": {
+        "sourceType": "module",
```

Comment:
I am not 100% sure on that but I think it would be more accurate to use the `@babel/eslint-parser` here and keep `sourceType` being `script` as that's what happens at runtime.

Reply:
Setting `@babel/eslint-parser` as the parser in the top-level config seems to works without having to specify those parserOptions

Reply:
Good point, I made the change.